api/scripts/process_tokens_from_old_transfers.py

```diff
@@ -0,0 +1,109 @@
+    database = Database(connectionString=databaseConnectionString)
+    saver = Saver(database=database)
+    retriever = Retriever(database=database)
+    s3manager = S3Manager(region='eu-west-1', accessKeyId=os.environ['AWS_KEY'], accessKeySecret=os.environ['AWS_SECRET'])
+    workQueue = SqsMessageQueue(region='eu-west-1', accessKeyId=os.environ['AWS_KEY'], accessKeySecret=os.environ['AWS_SECRET'], queueUrl='https://sqs.eu-west-1.amazonaws.com/097520841056/notd-work-queue')
+    tokenQueue = SqsMessageQueue(region='eu-west-1', accessKeyId=os.environ['AWS_KEY'], accessKeySecret=os.environ['AWS_SECRET'], queueUrl='https://sqs.eu-west-1.amazonaws.com/097520841056/notd-token-queue')
+    awsRequester = AwsRequester(accessKeyId=os.environ['AWS_KEY'], accessKeySecret=os.environ['AWS_SECRET'])
+    ethClient = RestEthClient(url='https://nd-foldvvlb25awde7kbqfvpgvrrm.ethereum.managedblockchain.eu-west-1.amazonaws.com', requester=awsRequester)
+    requester = Requester()
+    tokenMetadataProcessor = TokenMetadataProcessor(requester=requester, ethClient=ethClient, s3manager=s3manager, bucketName=os.environ['S3_BUCKET'])
+    openseaApiKey = os.environ['OPENSEA_API_KEY']
+    collectionProcessor = CollectionProcessor(requester=requester, ethClient=ethClient, openseaApiKey=openseaApiKey, s3manager=s3manager, bucketName=os.environ['S3_BUCKET'])
+    tokenManager = TokenManager(saver=saver, retriever=retriever, tokenQueue=tokenQueue, collectionProcessor=collectionProcessor, tokenMetadataProcessor=tokenMetadataProcessor)
+
+    await database.connect()
+    await workQueue.connect()
+    await s3manager.connect()
+    await tokenQueue.connect()
+    cache = set()
+    registryCache = set()
+    currentBlockNumber = startBlockNumber
+    while currentBlockNumber < endBlockNumber:
+        start = currentBlockNumber
+        end = min(currentBlockNumber + batchSize, endBlockNumber)
+        logging.info(f'Working on {start} to {end}...')
+        query = (
+             sqlalchemy.select(TokenTransfersTable.c.registryAddress, TokenTransfersTable.c.tokenId)
+             .where(TokenTransfersTable.c.blockNumber >= start)
+             .where(TokenTransfersTable.c.blockNumber < end)
+         )
+        result = await database.execute(query=query,)
+        tokensToProcess = set()
+        collectionsToProcess = set()
+        for (registryAddress, tokenId) in result:
+            if (registryAddress, tokenId) in cache:
+                continue
+            cache.add((registryAddress, tokenId))
+            tokensToProcess.add((registryAddress, tokenId))
+            if registryAddress in registryCache:
+                continue
+            registryCache.add(registryAddress)
+            collectionsToProcess.add(registryAddress)
+        print('len(tokensToProcess)', len(tokensToProcess))
+        print('len(collectionsToProcess)', len(collectionsToProcess))
+        for tokensToProcess in list_util.generate_chunks(lst=list(tokensToProcess), chunkSize= 50):
```

Comment:
you are overriding the `tokensToProcess` variable. change this to `tokensToProcessChunk`. Also it would be better to do the chunking inside the function you've created above instead of out here